src/routes/inference/+page.svelte

```diff
@@ -0,0 +1,305 @@
+	/>
+</svelte:head>
+
+<div class="container mx-auto px-10 my-10">
+	<div class="grid grid-cols-1 md:grid-cols-3 gap-4 lg:gap-10">
+		<div class="col-span-2">
+			<h1 class="text-4xl">{title}</h1>
+			<br /><br />
+			<p class="text-xl">
+				{description}
+			</p>
+			<br />
+			<a href="https://onnxruntime.ai/docs/build/inferencing.html" class="btn btn-primary"
+				>Learn how to build ONNX Runtime for inferencing →</a
+			>
+		</div>
+		<div class="m-auto">
+			<OnnxLight height={250} width={250} />
+		</div>
+	</div>
+</div>
+<div class="container mx-auto px-10 my-10">
+	<h1 class="text-4xl pb-4">Benefits</h1>
+	<div class="grid gap-10 grid-cols-1 md:grid-cols-3 pb-10">
+		<div class="card bg-base-300">
+			<div class="card-body items-center text-center">
+				<h2 class="card-title">Cost savings vs. running models in the cloud</h2>
+			</div>
+		</div>
+		<div class="card bg-base-300">
+			<div class="card-body items-center text-center">
+				<h2 class="card-title">Better latency and availability than request in the cloud</h2>
+			</div>
+		</div>
+		<div class="card bg-base-300">
+			<div class="card-body items-center text-center">
+				<h2 class="card-title">More privacy since data stays on device</h2>
+			</div>
+		</div>
+	</div>
+	<div class="grid gap-10 grid-cols-1 md:grid-cols-2 mx-auto">
+		<div class="card bg-base-300">
+			<div class="card-body items-center text-center">
+				<h2 class="card-title">
+					Easily enable cross-platform portability with the same implementation through the browser
+				</h2>
+			</div>
+		</div>
+		<div class="card bg-base-300">
+			<div class="card-body items-center text-center">
+				<h2 class="card-title">
+					Simplify the distribution experience without needing any additional libraries and driver
+					installations
+				</h2>
+			</div>
+		</div>
+	</div>
+</div>
+<div class="container mx-auto px-10 my-10">
+	<h1 class="text-4xl pb-4">Use Cases</h1>
+	<div class="grid gap-10 grid-cols-1 md:grid-cols-2 lg:grid-cols-4 mx-auto">
+		<div class="card bg-base-300">
+			<div class="card-body items-center text-center">
+				<h2 class="card-title">Improve inference performance for a wide variety of ML models</h2>
+			</div>
+		</div>
+		<div class="card bg-base-300">
+			<div class="card-body items-center text-center">
+				<h2 class="card-title">Run on different hardware and operating systems</h2>
+			</div>
+		</div>
+		<div class="card bg-base-300">
+			<div class="card-body items-center text-center">
+				<h2 class="card-title">Train in Python but deploy into a C#/C++/Java app</h2>
+			</div>
+		</div>
+		<div class="card bg-base-300">
+			<div class="card-body items-center text-center">
+				<h2 class="card-title">
+					Train and perform inference with models created in different frameworks
+				</h2>
+			</div>
+		</div>
+	</div>
+</div>
+<LandingHero
+	title="ONNX Runtime Mobile"
+	description="ONNX Runtime Mobile allows you to run model inferencing on mobile devices (iOS and Android)."
```

Comment:
ONNX Runtime mobile runs models on mobile devices using the same API used for cloud-based inferencing. Developers can use their mobile language and development environment of choice to add AI to Android, iOS, react-native, MAUI/Xamarin applications in Swift, Objective-C, Java, Kotlin, JavaScript, C, and C++.